app/controllers/concerns/metrics_concern.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+# Methods for logging usage metrics based on requests for files
+module MetricsConcern
+  def track_download(druid, file: nil)
```

Comment:
Can this be made Async so that it doesn't block downloads?

Reply:
For sure, this seems like a great use case for async. Should I just use the `async` gem? Never done it before in ruby.

Reply:
I was thinking ActiveJob.

Reply:
Updated to use a job